content/en/docs/reference/controllers/serviceaccount-token.md

```diff
@@ -0,0 +1,26 @@
+---
+title: ServiceAccount token controller
+content_template: templates/concept
+---
+
+{{% capture overview %}}
+
+One of a pair of controllers for the {{< glossary_tooltip text="ServiceAccount" term_id="service-account" >}}
+resource.
```

Comment:
remove `resource`?
A ServiceAccount Token Controller generates and assigns API access tokens for a ServiceAccount.